tutorials/btp-sdm-gwi-s4hana-maintain-businessroles/btp-sdm-gwi-s4hana-maintain-businessroles.md

```diff
@@ -0,0 +1,133 @@
+
+1. Logon to your SAP S/4HANA Cloud, public edition. Search for **Maintain Business Roles** app.
+
+    !![Selecting App](Select_Maintain_Business_Roles.png)
+
+2. Click on **New** at the bottom of the screen.
+
+    !![Create New Role](Click_New.png)
+
+3. In the **New Business Role** window, enter the following details:
+
+    |  Field Name     | Value
+    |  :------------- | :-------------
+    |  Business Role ID | **`FOR_TUTORIAL`**
+    |  Business Role Description | **`For tutorial purposes`**
+
+      !![Enter_Business_Role_Details](Enter_Business_Role_Details.jpg)
+
+4. Choose **Create**.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Add business catalog)]
+
+1. Navigate to **Assigned Business Catalogs** tab and select **Add**.
+
+    !![Assigned_Business_Catalog](Assigned_Business_Catalog.png)
+
+2. Search for `file management`, select `File Management - My Files` and click **Apply** and **OK**.
+
+    !![Look for file management](Search for File management.png)
+
+
+[DONE]
+[ACCORDION-END]
+
+
+[ACCORDION-BEGIN [Step 3: ](Maintain restrictions)]
+
+1. In the same business role, select **Maintain Restrictions**.
+
+    !![Maintain_Restrictions](Maintain_Restrictions.png)
+
+2. Select **Access Categories** from the left-side navigation menu and maintain the following values:
+
+    |  Field Name     | Value
+    |  :------------- | :-------------
+    |  Write, Read, Value Help | **`Restricted`**
+    |  Read, Value Help | **`Restricted`**
+
+    !![Accessing Categories from left side menu](Access_Categories.png)
+
+3. Select **Assigned Restriction Types** and choose `File Share Content`.
+
+    !![Access Restriction type](Assigned Restriction Type.png)
+
+4. In the **Restrictions and Values** section, enter the following details.  
+
+    |  Field Name     | Value
+    |  :------------- | :-------------
+    |  File Share Type | **`2`**
+    |  File Share ID | **`ZTEST`**
+    |  File Share Content Group | **`Unrestricted Access`**
+
+    >**Remember**: During the process of creating a communication system, you maintained **File Share ID** value. Please make sure the same **File Share ID** is being used.
+
+    File Share Type
+
+      !![File_Share_Type](File_Share_Type.png)
+
+    File Share ID
+
+      !![File_Share_ID](File_Share_ID.png)
+
+    File Share Content Group  
+
+      !![File_Share_Content](File_Share_Content Group.png)
+
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](Add business user)]
+
+1. Select **Assigned Business Users** and click **Add**.
+
+    !![AddbusinessUser](Add business user.png)
+
+2. Search for your **User Name**, select it and click **Apply** and **OK**.
+    >A username that you have maintained. Please select the valid username you have used in both SAP BTP and S/4HANA Cloud, public edition.
+
+    !![Select Username](Select Username.png)
+
+3. Choose **Save**.
+
+    !![Save](Save.png)
+
+[DONE]
+[ACCORDION-END]
+
+
+[ACCORDION-BEGIN [Step 5: ](Knowledge check)]
```

Comment:
All the other ones you said "Test yourself"

Reply:
corrected